styleguide/source/_patterns/03-templates/04-topic/_topic.scss

```diff
@@ -29,10 +29,15 @@
 }
 
 // Style the topic labels for each section consistently.
-.topic .title-label {
+*[class^="topic_"] .title-label {
   line-height: 1.38889;
   @include gutter($margin-bottom-half...);
-  @include gutter($padding-bottom-quarter...); //use quarter to offset line height of lable text
+  @include gutter($padding-bottom-quarter...); // use quarter to offset line height of label text.
+}
+
+.topic_related-articles .title-label,
+.topic_topic-articles .title-label,
+.topic_related-content_list .title-label {
```

Comment:
This is applying the and the label appears correctly underlined with the component is shown on the topic page (http://localhost:3000/?p=templates-topic) but not when the component is shown by itself (http://localhost:3000/?p=molecules-article-preview-small-image).